src/main/java/com/coravy/hudson/plugins/github/GithubProjectProperty.java

```diff
@@ -31,8 +30,18 @@
     private String projectUrl;
 
     @DataBoundConstructor
-    public GithubProjectProperty(String projectUrl) {
-        this.projectUrl = new GithubUrl(projectUrl).baseUrl();
+    public GithubProjectProperty(String url) {
+        this.projectUrl = new GithubUrl(url).baseUrl();
+    }
+
+    /**
+     * Same as {@link #getProjectUrl}, but with a property name and type
+     * which match those used in the {@link #GithubProjectProperty} constructor.
+     * Should have been called {@code getProjectUrl} and that method called something else
+     * (such as {@code getNormalizedProjectUrl}), but that cannot be done compatibly now.
+     */
+    public String getUrl() {
```

Comment:
I mean add javadoc that describes that this method returns full url to project. Because GH related code can deal with ghUrl as a plain api url, or url to project/repository. Simple 'url' withour javadoc doesn't provide good description.